db/seeds.rb

```diff
@@ -7,4 +7,4 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 #
 # Remember to ensure that the commands in this file are idempotent!
-Instance.find_or_create_by(name: 'Default', host: '*')
+Instance.find_or_create_by(name: 'Default', host: 'coursemology.org')
```

Comment:
Add the validation so the users cannot make a wrong instance, but the seed 
needs to be there as part of the initial data set.